spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/util/DockerComposeFactory.java

```diff
@@ -50,12 +50,12 @@
 	/**
 	 * Data Flow version to use for the tests.
 	 */
-	public static final String DEFAULT_DATAFLOW_VERSION = "2.5.0.BUILD-SNAPSHOT";
+	public static final String DEFAULT_DATAFLOW_VERSION = "2.6.0.BUILD-SNAPSHOT";
```

Comment:
These changes are already in. You might want to remove so that would help back-porting easier ;)